src/components/fields/_BasicCell/BasicCellContextMenuActions.tsx

```diff
@@ -5,56 +5,63 @@ import CopyCells from "@src/assets/icons/CopyCells";
 import Paste from "@mui/icons-material/ContentPaste";
 
 import { useProjectContext } from "@src/contexts/ProjectContext";
+import { useSnackbar } from "notistack";
+import { SelectedCell } from "@src/atoms/ContextMenu";
 
-export default function BasicContextMenuActions() {
-  const { contextMenuRef, tableState, deleteCell, updateCell } =
-    useProjectContext();
+export interface IContextMenuActions {
+  label: string;
+  icon: React.ReactNode;
+  onClick: () => void;
+}
```

Comment:
Note for the future: when we have more context menu actions for different field types, we should move this to `src/components/fields/types.ts`. I’m also not sure about having this file be inside `_BasicCell`. This is fine for now however.